cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/AbstractContext.java

```diff
@@ -217,11 +217,10 @@ protected AbstractContext(Program program, ProgramOptions programOptions, CConfi
 
   private MetricsCollectionService getMetricsService(CConfiguration cConf, MetricsCollectionService metricsService,
                                                      Map<String, String> runtimeArgs) {
-    String emitMetricsPreference = runtimeArgs.get(Constants.Metrics.EMIT_PROGRAM_CONTAINER_METRICS);
-
-    // Get the option for emitting metrics from preferences or as a fall back from cConf
-    boolean emitMetrics = emitMetricsPreference != null ? Boolean.valueOf(emitMetricsPreference) :
-      cConf.getBoolean(Constants.Metrics.EMIT_PROGRAM_CONTAINER_METRICS);
+    boolean emitMetrics = cConf.getBoolean(Constants.Metrics.EMIT_PROGRAM_CONTAINER_METRICS);
```

Comment:
Better just call the constants `PROGRAM_METRICS_ENABLED` to match it with the constant value.